stress/v2/ponyExpress/storeFront_test.go

```diff
@@ -0,0 +1,33 @@
+package ponyExpress
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	influx "github.com/influxdata/influxdb/client/v2"
+)
+
+func NewBlankTestPoint() *influx.Point {
```

Comment:
This is a utility to create blank points for testing the response path. Is there a naming convention for that? Should the function be unexported?